woocommerce/woocommerce
# CYS: Fix tooltip overlap with pattern

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR fixes the tooltip overall with the pattern preview. Futhermore, it removes `block-pattern-list.jsx` and uses the Gutenberg implementation for the header and footer patterns. 









| Before | After |
|--------|--------|
| <video src="https://github.com/user-attachments/assets/29f86cd7-a297-4456-b1ab-3730febcb747" /> | <video src="https://github.com/user-attachments/assets/dede0842-5b39-464b-a32f-741c1866d82c" /> | 




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Head over to WooCommerce > Home > Customize your store.
2. Click on "Design your Homepage".
3. Hover a pattern.
4. Ensure that the tooltip doesn't overlap the pattern.
5. Go back.
6. Click on "Choose your header".
7. Do a smoke test.
8. Go back.
9. Click on "Choose your footer".
10. Do a smoke test



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

CYS: Fix the tooltip overlap with the pattern.

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
